recipes-security/refpolicy/refpolicy-mcs-2.%/patches/policy.modules.kernel.files.diff

```diff
@@ -91,7 +91,15 @@
  ##	Read sockets with the default file type.
  ## </summary>
  ## <param name="domain">
-@@ -3246,6 +3267,36 @@ interface(`files_create_boot_flag',`
+@@ -2939,6 +2960,7 @@ interface(`files_read_etc_files',`
+ 
+ 	allow $1 etc_t:dir list_dir_perms;
+ 	read_files_pattern($1, etc_t, etc_t)
++	allow $1 etc_t:file map;
```

Comment:
Something is weird to be mmaping /config/etc/passwd...  Since as you say, upstream doesn't assign the perm in `files_read_etc_files()`

Reply:
One AVC that motivated that change is:
```
denied  { map } for  pid=971 comm="dbus-daemon" path="/config/etc/passwd" dev="dm-8" ino=1283 scontext=system_u:system_r:system_dbusd_t:s0-s0:c0.c10
23 tcontext=system_u:object_r:etc_t:s0 tclass=file permissive=1
```
Maybe something else is not transitioning to its own label?